tests/driver_mrf24j40/Makefile

```diff
@@ -0,0 +1,16 @@
+APPLICATION = driver_mrf24j40
```

Comment:
> Or did I miss any additional functionality provided by this test app?

Automated testing?

> Since (I believe) the gnrc_networking example also has a test on the link layer (the function called txtsnd) I agree.

#6397 removed that and fixed #5034 through that.